client/components/theme-preview-modal/style.scss

```diff
@@ -2,10 +2,8 @@
 @import "@wordpress/base-styles/mixins";
 @import "@wordpress/base-styles/variables";
 
-$break-design-preview: 1024px;
```

Comment:
Maybe there is a better place for this CSS variable @taipeicoder?

Reply:
This PR removes this custom breakpoint in favor of Gutenberg's, which is what we wanted to do 🙂